tests/Client/LogTest.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace HelloFresh\Stats\Client;
+
+
+use HelloFresh\Stats;
+use HelloFresh\Stats\Bucket;
+use HelloFresh\Stats\Bucket\MetricOperation;
+use HelloFresh\Stats\Bucket\Plain;
+use HelloFresh\Stats\Incrementer;
+use HelloFresh\Stats\State;
+use PHPUnit\Framework\TestCase;
+
+class LogTest extends TestCase
+{
+    public function setUp()
+    {
+        mt_srand(time());
```

Comment:
Why do we need this?

Reply:
I use `mt_rand` to generate test values, so need to seed it first